samples/bootloader/README.rst

```diff
@@ -218,12 +218,18 @@ Firmware versions using the |NSIB| are kept in the form of a *monotonic counter*
 Counter values are kept as slots in the flash memory, with each new counter value occupying a new slot.
 See :kconfig:option:`CONFIG_SB_MONOTONIC_COUNTER` for more details.
 
-To set the counter value of an application, build it using the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option:
+You can set the counter value for different elements in the bootloader sequence.
+Using :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option in ``prj.conf`` will allow you to change the version of the application image.
+However, in order to specify :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` for the MCUboot image you have to create a ``child_image`` folder in the project directory.
+For more information regarding the child image configuration refer to :ref:`ug_multi_image_build_scripts` section.
 
 .. code-block::
 
    CONFIG_FW_INFO_FIRMWARE_VERSION=<integer>
 
+.. note::
+   You can also set the counter value for the application upgrade, by using :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` option.
```

Comment:
I'm afraid this sentence can confuse more than it helps unless we add more details. As far as I understand, it's not for the application image, but the MCUboot image. I don't know the full details of it, but I think it's only a string value that you can access from the application and log to console or report to the cloud. This is typically used to verify that a device booted and reported the new version number after the upgrade completed. This is the help string from the kconfig definition:
> Value to be passed as 'version' argument to 'imgtool.py' when
	  creating signed image. Note that no semantics are connected to
	  this variable. It does not provide downgrade prevention, and is only
	  valuable for debugging purposes. Format: maj.min.rev+build with
	  latter parts optional.